app/controllers/api/v2/commodities_controller.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V2
     class CommoditiesController < ApiController
       before_action :find_commodity, only: %i[show changes]
-      before_action :set_meursing_additional_code, only: :show
+      around_action :configure_meursing_additional_code, only: :show
```

Comment:
I see that this is not just a simple set, but I think it's fine and clearer to keep the name as before: `set_meursing_additional_code`

Reply:
I'm actually following the same approach as the time machine, basically. I could extract this into a MuersingConfigurator or equivalent at some point